packages/clay-alert/src/index.tsx

```diff
@@ -61,7 +61,7 @@ export interface IClayAlertProps extends React.HTMLAttributes<HTMLDivElement> {
 	/**
 	 * Flag to indicate alert should automatically call `onClose`. It also
 	 * accepts a duration(in ms) which indicates how long to wait. If `true`
-	 * is passed in, the timeout will be 8000ms.
+	 * is passed in, the timeout will be 10000ms.
```

Comment:
While you're here, you may as well fix the typo on the line above (missing space before the `(`).